doc/whatsnew/fragments/3995.feature

```diff
@@ -0,0 +1,6 @@
+Support for ``NO_COLOR`` and ``FORCE_COLOR`` environment variables has been added.
+When running `pylint`, the reporter that reports to ``stdout`` will be modified according
+to the requested mode.
+The order is: ``NO_COLOR`` > ``FORCE_COLOR`` > ``--output=...``.
+
+Closes #3995 (https://github.com/pylint-dev/pylint/issues/3995).
```

Comment:
```suggestion
Closes #3995
```

Reply:
I have no opinion either way / Idk "how" this is rendered (neither where).

I simply prefer "links all the way", instead of having to do manual work "to see what #3995 is" (in a non-GH context)

Reply:
Since we tend to do this without links I would prefer that. The consistency makes it easier to parse these (if we ever want to) and attach the links for all of them all at once.

Reply:
There will be an issue with tbump parsing during release if we don't apply this so I'm applying.